mk/dependencies/go-deps.sh

```diff
@@ -12,6 +12,7 @@ PGKUMADOC=github.com/kumahq/protoc-gen-kumadoc@$(go list -f '{{.Version}}' -m gi
 GINKGO=github.com/onsi/ginkgo/v2/ginkgo@$(go list -f '{{.Version}}' -m github.com/onsi/ginkgo/v2)
 CONTROLLER_GEN=sigs.k8s.io/controller-tools/cmd/controller-gen@$(go list -f '{{.Version}}' -m sigs.k8s.io/controller-tools)
 
+> mk/dependencies/go-deps.versions
```

Comment:
Uhhh what does this mean? :grinning:

Reply:
This statement resets the file content to empty, identical to the following:
```
echo '' > mk/dependencies/go-deps.versions
```